.github/workflows/cppcheck.yml

```diff
@@ -0,0 +1,13 @@
+name: cppcheck-action
```

Comment:
This test fails.

Reply:
> This test fails.

It actually succeeds.

https://github.com/czm23333/curve/actions/runs/6058965735/job/16441596764

I have no idea why it fails here.